### PR TITLE
Make test_mpc compare MPC results and results in the clear

### PR DESCRIPTION
**Pull request by akoshelev**
Printing secret shared values is great, but humans are not trained to read them unfortunately.
It is probably better to print expected results and actual results received from MPC ring.
CLI should indicate whether there is a difference and highlight it.

```bash
+-----+----------+---------+-------+
| Row | Expected | Actual  | Diff? |
+==================================+
| 0   | Some(0)  | Some(0) |       |
|-----+----------+---------+-------|
| 1   | Some(3)  | Some(3) |       |
|-----+----------+---------+-------|
| 2   | Some(3)  | Some(3) |       |
+-----+----------+---------+-------+
```

it is not visible here, but matching rows are green and if there is a mismatch, it will show as red.

It also suppresses `test_mpc` verbose output in integration tests.

### CODE REVIEW

File: src/bin/test_mpc.rs [hunk 1]
```diff
@@ -83,24 +81,58 @@ async fn clients_ready(clients: &[MpcHelperClient; 3]) -> bool {
...
+    let mut actual = actual.into_iter().fuse();
+    let mut mismatch = Vec::new();
+
+    let mut table = Table::new();
+    table.set_header(vec!["Row", "Expected", "Actual", "Diff?"]);
+
+    let mut i = 0;
+    loop {
+        let next_expected = expected.next();
+        let next_actual = actual.next();
```
Comment (martinthomson) [inline src/bin/test_mpc.rs:100]:
```rust
for (next_expected, next_actual) = zip(expected, actual).filter(|(e, a)| e.is_some() || a.is_some()) {
  ...
}
```
?

**Review by martinthomson — APPROVED** [review]

File: src/bin/test_mpc.rs [hunk 2]
```diff
@@ -83,24 +81,57 @@ async fn clients_ready(clients: &[MpcHelperClient; 3]) -> bool {
...
+
+        match (&next_expected, &next_actual) {
+            (None, None) => break,
```
Comment (martinthomson) [inline src/bin/test_mpc.rs:103]: This is still better as:

```rust
if next_expected.is_none() && next_actual.is_none() {
  break;
}
```

File: src/bin/test_mpc.rs [hunk 3]
```diff
@@ -123,57 +153,80 @@ async fn main() -> Result<(), Box<dyn Error>> {
...
-                        input, &clients, query_id,
-                    )
-                    .await;
-                    print_output(&output);
-                }
-                InputType::Int64 => panic!("Only field values are supported"),
-            }
-        }
+        TestAction::Multiply => multiply(args, &clients).await,
+        TestAction::SemiHonestIPA => semi_honest_ipa(args, &clients).await,
```
Comment (martinthomson) [inline src/bin/test_mpc.rs:157]: No Malicious?

**Review by martinthomson — APPROVED** [review]
The approval stands :)

File: src/bin/test_mpc.rs (hunk 3 above)
Comment (akoshelev) [reply src/bin/test_mpc.rs:157]: would be trivial to add, just not there yet

File: src/bin/test_mpc.rs (hunk 2 above)
Comment (akoshelev) [reply src/bin/test_mpc.rs:103]: I don't mind doing it this way!

**Pull request merged**